addons/hw_drivers/tools/helpers.py

```diff
@@ -61,14 +63,14 @@ def run(self):
 elif platform.system() == 'Linux':
     @contextlib.contextmanager
     def writable():
-        subprocess.call(["sudo", "mount", "-o", "remount,rw", "/"])
-        subprocess.call(["sudo", "mount", "-o", "remount,rw", "/root_bypass_ramdisks/"])
         try:
+            subprocess.run(["sudo", "mount", "-o", "remount,rw", "/"], check=True)
+            subprocess.run(["sudo", "mount", "-o", "remount,rw", "/root_bypass_ramdisks/"], check=True)
             yield
         finally:
-            subprocess.call(["sudo", "mount", "-o", "remount,ro", "/"])
-            subprocess.call(["sudo", "mount", "-o", "remount,ro", "/root_bypass_ramdisks/"])
-            subprocess.call(["sudo", "mount", "-o", "remount,rw", "/root_bypass_ramdisks/etc/cups"])
+            subprocess.run(["sudo", "mount", "-o", "remount,ro", "/"], check=True)
+            subprocess.run(["sudo", "mount", "-o", "remount,ro", "/root_bypass_ramdisks/"], check=True)
+            subprocess.run(["sudo", "mount", "-o", "remount,rw", "/root_bypass_ramdisks/etc/cups"], check=True)
```

Comment:
check = true without `try / except`
Since the stderr and stdout are redirected to odoo_logs anyway we can keep it at "false" as it was before